File: JAVA/UberVan.java
import java.util.ArrayList;
import java.util.Map;

class UberVan extends Car {       /* extends means inheritance */
    Map<String, Map<String, Integer >> typeCarAccepted;      /* Map<brand, Map<model, year>>*/
    ArrayList<String> seatsMaterial;

    public UberVan(String license, Account driver, Map<String,
                Map<String, Integer >> typeCarAccepted, ArrayList<String> seatsMaterial) {
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}

File: PHP/UberVan.php
<?php
require_once('Car.php');

class UberVan extends Car {
    public $typeCarAccepted;
    public $typeMaterial;

    public function __construct($license, $driver, $typeCarAccepted, $typeMaterial) {
        parent::__construct($license, $driver);
        $this->typeCarAccepted = $typeAccepted;
        $this->typeMaterial = $typeMaterial;
    }
}

?>